dev-tools/cmd/index_template/index_template.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/template"
+)
+
+// Generates index templates for the beats
+//
+// The command which can be used in the Makefile looks as following
+//
+//   go run ${ES_BEATS}/dev-tools/cmd/index_template/index_template.go -files ${PWD}/${ES_BEATS}/libbeat/_meta/fields.generated.yml,${BEAT_GOPATH}/src/${BEAT_PATH}/_meta/fields.yml,${BEAT_GOPATH}/src/${BEAT_PATH}/_meta/fields.generated.yml -es-version 5.0.0 -beatname ${BEAT_NAME} -output ${BEAT_GOPATH}/src/${BEAT_PATH}/${BEAT_NAME}.template.json
+
+func main() {
+
+	version := flag.String("es-version", "", "Elasticsearch version")
```

Comment:
It would be nice to say what the default value is here. Could you call `beat.GetDefaultVersion()` here to populate the default value so that it shows up in the help.